src/main/java/org/isf/disease/rest/DiseaseController.java

```diff
@@ -247,7 +247,7 @@ public DiseaseController(DiseaseBrowserManager diseaseManager, DiseaseMapper dis
 			throw new OHAPIException(new OHExceptionMessage(null, "duplicated disease description for the same disease type", OHSeverityLevel.ERROR), HttpStatus.INTERNAL_SERVER_ERROR);
 		}
 		
-		if(diseaseManager.newDisease(disease)) {
+		if(diseaseManager.newDisease(disease) != null) {
```

Comment:
I know that you are modifying an existing line but while you are there it would be good to change `if(` to `if (` as was done in the other PR.  Thanks.